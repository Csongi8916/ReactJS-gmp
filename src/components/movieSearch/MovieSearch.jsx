import React from 'react';
import Logo from '../Logo/Logo';
import MovieSearchControls from './MovieSearchControls/MovieSearchControls';

import style from './MovieSearch.scss';

export default function MovieSearch() {
  return (
    <div className="MovSearch">
      <div className="MovSearch_Logo">
        <Logo />
      </div>
      <div className="MovSearch_Content">
        <MovieSearchControls />
      </div>
    </div>
  )
}
