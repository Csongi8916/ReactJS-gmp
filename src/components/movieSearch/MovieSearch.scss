$nrRedColor: #f65261;
$nrBlackColor: #232323;
$nrWhiteColor: #ffffff;

.MovSearch {
  display: flex;
  flex-direction: column;
  background-color: $nrBlackColor;
  padding-top: 20px;
  padding-bottom: 60px;
}

.MovSearch_Logo {
  margin-left: 4vw;
}

.MovSearch_Content {
  margin-left: 7vw;
  margin-right: 7vw;
}