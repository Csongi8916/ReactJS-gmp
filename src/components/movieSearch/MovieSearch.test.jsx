import React from 'react';
import renderer from 'react-test-renderer';
import MovieSearch from './MovieSearch';
import { configure, shallow, mount, render } from 'enzyme';


test('Link changes the class when hovered', () => {
  const component = renderer.create(
    <MovieSearch />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('Link changes the class when hovered', () => {
  //MovSearchControl_Input
  const wrap = mount(<MovieSearch />);
  const input = wrap.find('.MovSearchControl_Input');
  input.simulate('change', { target: { value: 'abc' } });
  debugger;
  expect(true).toBe(true);
});