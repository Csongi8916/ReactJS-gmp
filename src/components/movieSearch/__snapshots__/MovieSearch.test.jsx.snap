// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<div
  className="MovSearch"
>
  <div
    className="MovSearch_Logo"
  >
    <span
      className="root"
    >
      <span
        className="main-part"
      >
        netflix
      </span>
      <span>
        rolulette
      </span>
    </span>
  </div>
  <div
    className="MovSearch_Content"
  >
    <div
      className="MovSearchControl"
    >
      <div>
        <p
          className="MovSearchControl_SearchTitle"
        >
          Find your movie
        </p>
        <input
          className="MovSearchControl_Input"
          onChange={[Function]}
          type="text"
        />
        <button
          className="NrxBtn"
          onClick={[Function]}
        >
          Search
        </button>
      </div>
      <div
        className="MovSearchControl_SearchMode"
      >
        <div
          className="NxrToogle"
        >
          <span
            className="NxrToogle_ModeTitle"
          >
            Search by
          </span>
          <div
            className="NxrToogle_Container"
          >
            <div
              className="NxrToogle_Btn NxrToogle_Btn-Active"
              onClick={[Function]}
            >
              title
            </div>
            <div
              className="NxrToogle_Btn NxrToogle_Btn-Inactive"
              onClick={[Function]}
            >
              gengre
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
